src/move.c

```diff
@@ -223,17 +208,19 @@ void move_front(void)
  */
 void move_left(void)
 {
-	move_straight_stop_middle();
-	turn_left_static();
-	move_straight_out_of_cell();
+	decelerate(current_cell_start_micrometers, 0.04, 0.335);
```

Comment:
Why that values? why not a macro? or a note on comments?

Reply:
#142 :innocent: